cmd/down.go

```diff
@@ -44,6 +46,14 @@ func Down() *cobra.Command {
 
 			log.Success("Development environment deactivated")
 
+			if rm {
+				if err := cleanVolume(dev); err != nil {
```

Comment:
we should create a separate event for this specifically, so we know if the clean fails vs the down operation.